### ISSUE TITLE
[ER][Accessibility] Landing page URL Prominance

### ISSUE THREAD

**Issue by stefnestor** [elastic/esdiag#164]
Porting from [Slack#esdiag](https://elastic.slack.com/archives/C07CQE98MBQ/p1754331532775739), for internal user accessibility the URLs/links folks should care about most (not just technical FYI's) should be most prominent 

- potentially report Kibana's URL instead of Elasticsearch's here 
    -  <img width="888" height="378" alt="Image" src="https://github.com/user-attachments/assets/2cbc3aba-ba65-4b38-a02e-3b7aacf934b3" />
    - If users copy-paste the Elasticsearch URL, they get faced with a native realm login (expected) which they cannot use since they need to use Kibana's OKTA login. 
- hidden non-obvious functionality is that the top-right Kibana icon is clickable but appears to only be hoverable (like the user icon towards its right side)
    - <img width="703" height="101" alt="Image" src="https://github.com/user-attachments/assets/9cf3ad64-dd04-4119-aced-2d53b0bc8288" />

-

**Comment by VimCommando** [comment]
I see what you're saying here. I'm moving where this is indicated in my next major branch.

Maybe this less prominent location will make it less confusing?

<img width="2988" height="1694" alt="Image" src="https://github.com/user-attachments/assets/2643d647-8c99-4bf3-8905-9f96800edc46" />
